test/workflows/components/workflows.libsonnet

```diff
@@ -196,60 +196,69 @@
                   template: "checkout",
                 }],
                 [
+                  //{
+                  //  name: "unit-test",
+                  //  template: "unit-test",
+                  //},
+                  //{
+                  //  name: "sdk-test",
+                  //  template: "sdk-test",
+                  //},
+                  //{
+                  //  name: "pylint-checking",
+                  //  template: "pylint-checking",
+                  //},
                   {
-                    name: "unit-test",
-                    template: "unit-test",
-                  },
-                  {
-                    name: "sdk-test",
-                    template: "sdk-test",
-                  },
-                  {
-                    name: "pylint-checking",
-                    template: "pylint-checking",
+                    name: "setup-cluster",
+                    template: "setup-cluster",
                   },
                 ],
+                //[
+                //  {
+                //    name: "build-kfserving-manager",
+                //    template: "build-kfserving",
+                //  },
+                //  {
+                //    name: "build-alibi-explainer",
+                //    template: "build-alibi-explainer",
+                //  },
+                //  {
+                //    name: "build-storage-initializer",
+                //    template: "build-storage-initializer",
+                //  },
+                //  {
+                //    name: "build-xgbserver",
+                //    template: "build-xgbserver",
+                //  },
+                //  {
+                //    name: "build-kfserving-executor",
+                //    template: "build-kfserving-executor",
+                //  },
+                //  {
+                //    name: "build-pytorchserver",
+                //    template: "build-pytorchserver",
+                //  },
+                //  {
+                //    name: "build-sklearnserver",
+                //    template: "build-sklearnserver",
+                //  },
+                //],
```

Comment:
remove commented code

Reply:
Yes, I will remove this later, I found there are some problems now to start KFService in the testing env, I commented out for quick debugging. :-)